docs/index.html

```diff
@@ -188,6 +189,12 @@ <h3>Built with Open Source</h3>
     // Add scale control
     map.addControl(new maplibregl.ScaleControl(), 'bottom-right');
 
+    // Add terrain control for 3D visualization
+    map.addControl(new maplibregl.TerrainControl({
+      source: 'mapterhorn-terrain',
+      exaggeration: 1.5
```

Comment:
Exaggeration should be 1.0. That means we can drop exaggeration property because the default value of the exaggeration is 1.0.